packages/governance/src/contractHelper.js

```diff
@@ -23,6 +28,9 @@ const publicMixinAPI = harden({
   getNat: M.call().returns(M.bigint()),
   getRatio: M.call().returns(M.record()),
   getString: M.call().returns(M.string()),
+  getTimerService: M.call().returns(TimerServiceShape),
+  getTimestamp: M.call().returns(TimestampShape),
+  getRelativeTime: M.call().returns(RelativeTimeShape),
```

Comment:
the types are narrower:
```suggestion
  getTimestamp: M.call().returns(TimestampRecordShape),
  getRelativeTime: M.call().returns(RelativeTimeRecordShape),
```

Reply:
done